### ISSUE TITLE
Unable to disable Vertical Whitespace rule

### ISSUE THREAD

**Issue by aquarius** [realm/SwiftLint#906]
I'm unable to disable the Vertical Whitespace rule (we use 2 newlines between extensions) in our project. This is our config file:

```disabled_rules: # rule identifiers to exclude from running
  - line_length
  - nesting
  - conditional_binding_cascade
  - vertical_whitespace
opt_in_rules:
  - empty_count

use_nested_configs: true
```

When I run `swiftline rules` it's correctly displayed as disabled:
`| vertical_whitespace            | no     | yes         | no                     | warning`

Any ideas how I can solve this?

**Comment by jpsim** [comment]
Your config file is missing the `disabled_rules:` entry for the rules you want to disable. Here's a minimal scenario that demonstrates `vertical_whitespace` being disabled:

```shell
$ tree -a
.
├── .swiftlint.yml
└── test.swift

0 directories, 2 files
$ cat .swiftlint.yml
disabled_rules:
  - vertical_whitespace
$ cat test.swift
let abc = 0


let def = 1
$ rm .swiftlint.yml
$ swiftlint
Linting Swift files in current working directory
Linting 'test.swift' (1/1)
test.swift:3: warning: Vertical Whitespace Violation: Limit vertical whitespace to a single empty line. Currently 2. (vertical_whitespace)
Done linting! Found 1 violation, 0 serious in 1 file.
```

**Comment by aquarius** [comment]
Thank you for your reply. The `disabled_rules:` line sadly got lost during copy and paste.

I figured out the reason for my issue. We have a `.swiftlint.yml` file in a folder with test cases. When I also disable the rule in this config file, it works as expected although the linted files aren't inside this subfolder.

I'm sadly unable to produce a sample for this.
